core/README.md

```diff
@@ -1,5 +1,5 @@
 # Trezor Core
 
-Firmware currently running on Model T.
+Firmware currently running on Trezor Safe 5, Safe 3 and Model T.
```

Comment:
```suggestion
Firmware currently running on Model T and the Trezor Safe family.
```
so that we don't have to touch this for a while? :)

Reply:
better, thanks. Can someone take over? 🙏🏻